packages/v4/src/content/design-guidelines/usage-and-behavior/wizard/wizard.md

```diff
@@ -122,13 +121,13 @@ A progressive wizard takes the same form as the standard wizard or wizard with s
 
 Here is an example...
 
-![progressive wizard step 1](./img/wizard-progressive-step1.png)
+<img src="./img/wizard-progressive-step1.png" alt="Wizard progressive step 1" width="990"/>
```

Comment:
Can you add a line between the images and the "Step ....." text right after it - right now they're showing up on the same line:
![image](https://user-images.githubusercontent.com/51165119/89949667-48521c80-dbf6-11ea-9b34-864b7e4f1094.png)